app/javascript/react/components/molecules/CalendarStationHeader/CalendarStationHeader.tsx

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+import { ActionButton } from "../../ActionButton/ActionButton.style";
+import { Button } from "../../Button/Button";
+import { StationValueLabel } from "../StationValueLabel";
+import * as S from "./CalendarStationHeader.style";
+import bellAlert from "../../../assets/icons/bellAlert.svg";
+import shareLink from "../../../assets/icons/shareLink.svg";
+import downloadImage from "../../../assets/icons/download.svg";
+import copyLink from "../../../assets/icons/copyLink.svg";
+
+interface CalendarStation {
+  stationName: string;
+  profile: string;
+  sensor: string;
+  lastUpdate: string;
+  streamData: StreamData;
+}
+
+interface StreamData {
+  day: string;
+  value: number;
+  parameter: string;
+}
+
+const CalendarHeaderName: React.FC<{ stationName: string }> = ({
+  stationName,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <S.HorizontalContainer>
+        <S.Description>{t("calendarHeader.stationPrefix")}</S.Description>
+        <S.Header>{stationName}</S.Header>
+      </S.HorizontalContainer>
+    </>
```

Comment:
`<> </>` Not needed, you only need it if you want to return two siblings next to one another - then a wrapper like this is crucial as react always want to return one outer element.